src/main/java/com/mixpanel/android/mpmetrics/InAppFragment.java

```diff
@@ -250,6 +258,28 @@ public float getInterpolation(float t) {
         }
     }
 
+    @SuppressWarnings("deprecation")
+    private void showLightStyle() {
+        final TextView titleView = (TextView) mInAppView.findViewById(R.id.com_mixpanel_android_notification_title);
+        final ImageView notifImage = (ImageView) mInAppView.findViewById(R.id.com_mixpanel_android_notification_image);
+
+        GradientDrawable viewBackground = new GradientDrawable();
+        viewBackground.setColor(0xFFFFFFFF);
+        viewBackground.setStroke(2, Color.rgb(218, 223, 232));
+        viewBackground.setCornerRadius(6);
+        if(Build.VERSION.SDK_INT < Build.VERSION_CODES.JELLY_BEAN) {
+            mInAppView.setBackgroundDrawable(viewBackground);
+        } else {
+            mInAppView.setBackground(viewBackground);
+        }
+        titleView.setTextColor(Color.rgb(123, 146, 163));
```

Comment:
Can we have this colors in `colors.xml` (`123, 146, 163` and `218, 223, 232`) and load them from there?